src/consts.js

```diff
@@ -12,6 +12,11 @@ const {
  */
 const APIFY_API_BASE_URL = 'https://api.apify.com';
 
+const SORT_BY = {
```

Comment:
You can use const from client as here https://github.com/apify/keboola-ex-apify/pull/53/files#diff-dbce482fef350b0dd7b7eea37fbb5aa11fe0b3b09260db653f1031bc82503a6cR1